cli/dev.js

```diff
@@ -3,14 +3,56 @@ import fs from 'fs'
 import { debounce } from 'lodash'
 import path from 'path'
 
-import { compileSchema } from './compile'
+import { compileSchema, compileTs } from './compile'
 import {
     copyByExtension,
     copyByExtensionWithExtensionReplacement,
 } from './copy'
 import * as paths from './paths'
+import { existsSyncWithExtension } from './utils'
 
-if (fs.existsSync(paths.app.mouldDirectory)) {
+;(async function dev() {
+    if (fs.existsSync(paths.app.mouldDirectory)) {
+        try {
+            await build()
```

Comment:
I don't run `build.js` file, because the build process for `dev` differs

For `dev`, the Mould Schema (`.mould`):
* might be missing if a user has just started working with Mould 
* should be compiled into React components so that user's application can import Mould generated components

For `build`, the Mould Schema (`.mould`):
* should be present to compile it